CoSA.py

```diff
@@ -236,15 +238,15 @@ def run_verification(config):
         with open(config.translate, "w") as f:
             f.write(printer.print_hts(hts, properties))
 
-    if config.simulate:
+    if config.simulate or config.sim_no_unroll:
```

Comment:
I would see the simulation without unrolling as a parameter for the simulation instead of a separate analsysis